erpnext/crm/doctype/appointment/appointment.py

```diff
@@ -6,7 +6,9 @@
 
 import frappe
 from frappe import _
+from frappe.desk.form.assign_to import add as add_assignment
 from frappe.model.document import Document
+from frappe.share import add
```

Comment:
`add` isn't intuitive. either import `frappe.share` and use `frappe.share.add` or import `add` as `add_share`.